DrinkingPub/Table.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace Vsite.Oom.DrinkingPub
+{
+    public class Table
+    {
+
+        public double TotalCost { get; private set; }
+        public int TableId { get; }
+        public string Name { get; }
+        public List<Order> TableOrders { get; }
```

Comment:
Ovo svojstvo vraća referencu na interni objekt, tako da se preko tog svojstva (iako ima samo metodu `get`), može promijeniti sadržaj rječnika izvan klase Table. To možete provjeriti tako da u metodi `Main` dodate naredbu:
`table1.TableOrders.Add(new Order());`
Da bi se onemogućile promjene izvan klase, svojstvo se može definirati kao:
`public IEnumerable<Order> Orders => tableOrders;`
s time da onda treba definirati privatno polje:
`private List<Order>tableOrders = new();`